notebooks/MAST/HSC/CASJOBS/HCV_CASJOBS/HCV_casjobs_demo.ipynb

```diff
@@ -0,0 +1,995 @@
+    "\n",
+    "Data tables in [MAST CasJobs](https://mastweb.stsci.edu/hcasjobs) are queried from Python using the [mastcasjobs](https://github.com/rlwastro/mastcasjobs) module. For similar examples using the [MAST API](https://catalogs.mast.stsci.edu/docs/hsc.html), which is easier to use but less powerful than CasJobs, see [HCV_API_demo](../HCV_API/HCV_API_demo.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Instructions: \n",
+    "* Complete the initialization steps [described below](#initialization).\n",
+    "* Run the notebook to completion. \n",
+    "* Modify and rerun any sections of the Table of Contents below.\n",
+    "\n",
+    "# Table of Contents\n",
+    "* [Intialization](#initialization)\n",
+    "* [Variable objects in IC 1613](#ic1613)\n",
+    "    * [Name resolver](#resolver)\n",
+    "    * [Select objects from HCV](#summary)\n",
+    "    * [Information on HCV variable classification](#classification)\n",
+    "    * [Sky coverage](#positions)\n",
+    "    * [Properties of variable objects](#variability)\n",
+    "    * [Color magnitude diagram](#cmd)\n",
+    "* [Light curve for a nova in M87](#m87)\n",
+    "    * [Extract and plot light curve for the nova](#lightcurve)\n",
+    "    * [HLA cutout images for selected measurements](#cutouts)\n",
+    "* [Compare the HCV automatic classification to expert validations](#expert)\n",
+    "    * [Plot MAD variability index distribution](#mad_expert)\n",
+    "    * [Plot fraction of artifacts vs. MAD](#artifacts)\n",
+    "* [Plot light curve for most variable high-quality candidate in the HCV](#most_variable)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Initialization <a name=\"initialization\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Install Python modules\n",
+    "\n",
+    "1. _This notebook requires the use of **Python 3**._\n",
+    "1. Modules can be installed with `conda`, if using the [Anaconda distribution][1] of python, or with `pip`.\n",
+    "   - If you are using `conda`, do not install / update / remove a module with `pip`, that exists in a `conda` [channel][1].\n",
+    "   - If a module is not available with `conda`, then it's okay to install it with `pip`\n",
+    "1. Use the following instructions to install / upgrade to use the correct version of `casjobs` and `mastcasjobs`.\n",
+    "\n",
+    "```\n",
+    "pip install git+git://github.com/dfm/casjobs@master\n",
+    "pip install git+git://github.com/rlwastro/mastcasjobs@master\n",
```

Comment:
This is potentially a bit problematic, because it's for a MAST service, but on a personal fork.  Do we think we can either 1) get `mastcasjobs` released so that it's pip-installable, or 2) move it to a `spacetelescope/mastcasjobs` repo?

Reply:
(the `dfm/casjobs` one has potentially the same issue but I am ignoring because it's not someone who's actually at the institute, but it's been stable for a long time so I don't expect it needs a ton of upkeep)

Reply:
This is something for @rlwastro to work one.

Reply:
OK, `mastcasjobs` is now installable from pip.  I also got Dan Foreman-Mackey to add permissions on pypi for the `casjobs` module, and I installed a new version of `casjobs` (0.0.2) on pypi as well.  The requirements are written correctly for `mastcasjobs` (`casjobs>=0.0.2`)  So now a single installation command will get them both:

`pip install mastcasjobs`

Reply:
@rlwastro To clarify, I can remove those two lines, because the correct packages will install based on the environment.yml file?